src/components/Rating/README.md

```diff
@@ -0,0 +1,4 @@
+## Rating
```

Comment:
Add an extended README.
You can see how the rest of the components have their README.
Button for instance